crates/frontend/src/components/experiment_form.rs

```diff
@@ -210,6 +230,44 @@ pub fn experiment_form(
                 metrics=metrics_rws.get_untracked()
                 on_change=Callback::new(move |metrics| metrics_rws.set(metrics))
             />
+
+            {move || {
+                let mut experiment_options: Vec<Option<ExperimentGroup>> = experiment_groups
+                    .iter()
+                    .map(|group| Some(group.clone()))
+                    .collect();
+                experiment_options.insert(0, None);
```

Comment:
this way you wont need to derive PartialEq on ExperimentGroup, as you dont even need to store the entire experiment group, only the group id is needed
```suggestion
                let mut experiment_options: Vec<Option<i64>> = experiment_groups
                    .iter()
                    .map(|group| Some(group.id.clone()))
                    .collect();
                experiment_options.insert(0, None);
```

Reply:
not a good idea, name is not unique, you will have collisions